.github/workflows/deploy.yml

```
@@ -26,4 +26,4 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs/book
```

```suggestion
          publish_dir: ./docs/book/html
```

And remove deploy-docs.yml again ..